src/main/java/org/opensearch/timeseries/model/Config.java

```diff
@@ -559,4 +562,14 @@ protected void checkAndThrowValidationErrors(ValidationAspect validationAspect)
             throw new TimeSeriesException(CommonMessages.FAIL_TO_VALIDATE);
         }
     }
+
+    public static Config parseConfig(Class<? extends Config> configClass, XContentParser parser) throws IOException {
```

Comment:
I see this hasn't been used yet, I assume you'll use it later on?

Reply:
yes